01week/resume/JavaScript/Javascript.js

```diff
@@ -0,0 +1,71 @@
+function sectionSetup(_selectedButton) {
+	var article_file = "./html/" + _selectedButton + ".html";
+  var sidebar_file = "./html/" + _selectedButton + "SideBar.html";
+	modifyArticle(article_file);
+	modifySideBar(sidebar_file);
+}
+
+function modifyArticle(_currentArticle){
+    var xhr = typeof XMLHttpRequest != 'undefined' ? new XMLHttpRequest() : new ActiveXObject('Microsoft.XMLHTTP');
```

Comment:
`!==` instead of `!=` especially if you dealing with undefined

also`'undefined'` is different then `undefined`